src/packaging/_musllinux.py

```diff
@@ -63,6 +63,10 @@ def platform_tags(arch: str) -> Iterator[str]:
     sys_musl = _get_musl_version(sys.executable)
     if sys_musl is None:  # Python not dynamically linked against musl.
         return
+    if arch == "armv8l":
+        # armv8l wheels are not accepted on PyPI
+        # The armv7l wheels can be installed.
+        arch = "armv7l"
```

Comment:
There's no `_have_compatible_abi` on musllinux and here, we just assume it's indeed armhf ABI.

Reply:
Any reason not to add one?

Reply:
It might be a good thing to add one, same as manylinux, but this is orthogonal to this PR.